modules/ROOT/nav.adoc

```diff
@@ -3,6 +3,7 @@
 * xref:first-login.adoc[First Login]
 * xref:upgrades.adoc[Upgrades and Rollbacks]
 * xref:silverblue-guide.adoc[Silverblue Guide]
+* xref:silverblue-flatpak-guide.adoc[Silverblue Flatpak Guide]
```

Comment:
As a general comment, I don't think it is right to accumulate multiple comprehensive user guides next to each other - these should be merged and made to fit into the overall documentation.

Reply:
Okay, I merged flatpak guide into the overall silverblue guide and removed the separate file. Should be good to go now.